test/e2e/tests/phishing-controller/phishing-detection.spec.js

```diff
@@ -315,6 +315,88 @@ describe('Phishing Detection', function () {
     );
   });
 
+  it('should block a website that makes a WebSocket connection to a malicious c2 site', async function () {
```

Comment:
nit: I don't think most people know what `c2` means (I didn't). Do you mind spelling things out? While acronyms can save time, that’s only true for VERY common acronyms, and maybe the person writing them. They generally actually increase the time it takes someone to read and understand the message.

Reply:
fixed! thank you!